main.go

```diff
@@ -41,7 +41,10 @@ func main() {
 
 	// Parse users and limit
 	users := strings.Split(userslist, ",")
-	limit := strings.Split(limitslist, ",")
```

Comment:
I don't see an error here!
If 'limitslist' is empty the 'strings.Split(limitslist, ",")' return empty []string slice

Reply:
Look issue 39806 in Go. I see this bug at me. I don't set limit, but len(limit) == 1, but slice empty.
Windows 10, Go 1.8.0, and 1.8.1

Reply:
OK :-)
Please look my new comments and fix it
Thank you! :-)